profiles/package.mask

```diff
@@ -1,6 +1,7 @@
 # Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (20 Dec 2020)
 # Masked for removal, as there is a newer version available
 =dev-haskell/cabal-install-3.0.0.0
+=dev-haskell/hackage-security-0.5.3.0-r3
```

Comment:
No need to mask outdated versions if there is nothing actively broken about the. We can just delete versions after some time (say, 2 weeks).